clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -582,7 +582,6 @@ static KafkaAdminClient createInternal(AdminClientConfig config,
         Metrics metrics = null;
         String clientId = generateClientId(config);
         Optional<ClientTelemetryReporter> clientTelemetryReporter = CommonClientConfigs.telemetryReporter(clientId, config);
-        
```

Comment:
?

Reply:
Not sure what's going on, I compared this with trunk and set it to that.

Reply:
🤷